mern/client/src/utils/QA.js

```diff
@@ -1,42 +1,29 @@
 import axios from "axios";
 
-var allQuestionList = [];
-
-export default class Parser {
-  parse(transcript) {
-    const startingList = [];
-
-    if (allQuestionList !== []) {
-      allQuestionList = [];
-    }
-
-    for (var i = 0; i < transcript.length; i++) {
-      this.createQAs(transcript[i]);
-      startingList.push(this.initialQAID(transcript[i]));
-    }
-
-    return { startingList, allQuestionList };
-  }
-
-  //Loops through all questions in the JSON object qlist, making them into objects in the databse
-  async createQAs(qlist) {
-    //qlist is a list of JSON objects that exists as questions/answer pairs
-
-    for (var i = 0; i < qlist.length; i++) {
-      try {
-        await axios.post("http://localhost:5000/qa/add", qlist[i]);
-        allQuestionList.push(qlist[i].id);
-      } catch (e) {
-        console.log(e.response.data);
-      }
+export default class QA {
```

Comment:
This isn't necessarily clean practice. This has the whole use case in one file when we should be separating out the interactor and controller and DAO into different folders.